package.json

```diff
@@ -23,11 +23,11 @@
     "dist-custom-elements": "npm run build-custom-elements && shx rm -rf dist/all/dist && shx cp -r output/dist dist/all/"
   },
   "peerDependencies": {
-    "@dojo/core": "~0.4.0",
+    "@dojo/core": "~0.5.0",
     "@dojo/has": "~0.1.1",
     "@dojo/i18n": "~0.5.1",
     "@dojo/shim": "~0.2.6",
-    "@dojo/widget-core": "~0.9.5"
+    "@dojo/widget-core": "./../widget-core/dist/dojo-widget-core-0.9.7-rc.1.tgz"
```

Comment:
Just remember to update this with the real release version before merging this.

Reply:
Of course, I do put it in the description that it requires releases and wouldn't merge if it doesn't pass CI 😄